package.json

```diff
@@ -34,8 +34,6 @@
     "prelint": "make build-package",
     "lint": "gulp lint",
     "test": "make test",
-    "test-pa11y": "make test-runner-pa11y",
-    "test-jest": "make test-runner-jest",
     "build": "make build",
```

Comment:
I guess I've never looked inside this file, but it seems very weird to me that the npm tasks call out to the Makefile... seems like most of the pipeline happens in JS except the jekyll build... make in a separate/future PR we can invert this?

Reply:
> I guess I've never looked inside this file, but it seems very weird to me that the npm tasks call out to the Makefile... seems like most of the pipeline happens in JS except the jekyll build... make in a separate/future PR we can invert this?

Yeah, it's a bit nonconventional as things stand. Also means there is no `make run` like there is in other projects.

I feel like `Makefile` and npm scripts serve many of the same goals, so maybe we could just get rid of one or the other. 🤷